scripts/deploy_schemas_to_node.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -m
+
+#Useful for future debugging
+npm --version
+ldd --version
+whoami
+
+/frequency/frequency --dev \
+    -lruntime=debug \
+    --instant-sealing \
+    --wasm-execution=compiled \
+    --execution=wasm \
```

Comment:
Side note, this these options will eventually be going away, but they shouldn't cause any issues.